lib/models/file-info.js

```diff
@@ -95,7 +97,33 @@ var FileInfo = CoreObject.extend({
     this._fileContents = fse.readFileSync(fullPath, { encoding: 'utf8' });
   },
 
+  updateImports: function() {
+    if (this.ext !== '.js' || !this._fileContents) { return; } // only process JavaScript files
+
+    var appName = this.options.projectName;
+
+    try {
+      var newContents = importDeclarationsTransform(
+        { source: this._fileContents,
+          fileInfo: this,
+          appName: appName,
+          fileInfos: this._fileInfoCollection._fileInfos
+        },
+        { jscodeshift: jscodeshift });
```

Comment:
This project is still in its infancy, and Node 0.12 has been out of support for > 6 months now.  I will happily release with the proper version bump (indicating the engine requirements have changed).